rdtools/system_analysis.py

```diff
@@ -0,0 +1,586 @@
+'''
+This module contains functions and classes for object-oriented end-to-end analysis
+'''
+import pvlib
+import pandas as pd
+import matplotlib.pyplot as plt
+from . import normalization
+from . import filtering
+from . import aggregation
+from . import degradation
+from . import soiling
+from . import clearsky_temperature
+from . import plotting
+
+
+class system_analysis():
+    '''
+    Class for end-to-end analysis
```

Comment:
Recommend moving __init__ docstring up to the class docstring such that help(system_analysis) and print(system_analysis.__doc__) presents useful information.  This is considered best practice per google:
https://stackoverflow.com/questions/37019744/is-there-a-consensus-what-should-be-documented-in-the-classes-and-init-docst

Reply:
Done, thanks!